src/test/java/uk/gov/pay/ledger/transaction/resource/TransactionResourceIT.java

```diff
@@ -717,7 +717,107 @@ public void shouldGetAllTransactionsAsCSVWithAcceptType() throws IOException {
                 .contentType("text/csv")
                 .extract().asInputStream();
 
-        assertHealthyCsvResponse(csvResponseStream, transactionFixture);
+        List<CSVRecord> csvRecords = CSVParser.parse(csvResponseStream, Charset.defaultCharset(), RFC4180.withFirstRecordAsHeader()).getRecords();
```

Comment:
I know you just copied this from what was already there but `Charset.defaultCharset()` almost certainly wants to be `StandardCharsets.UTF_8` (since that’s what we should be outputting from the transaction resource).

Reply:
cool, will change

Reply:
done